docs/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Functions - Strapi Developer Documentation
+description: 
+---
+
+<!-- TODO: update SEO -->
+
+# Functions
+
+The `./config/functions/` folder contains a set of JavaScript files in order to add dynamic and logic based configurations.
+
+All functions that are exposed in this folder are accessible via `strapi.config.functions['fileName']();`
```

Comment:
It's possible this might be `strapi.server.config.....` but again should check with Alex